.github/workflows/sgx-report.yml

```diff
@@ -0,0 +1,58 @@
+name: Create New SGX Report
+on: push
+jobs:
+  build_and_generate_report:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./notary-server
+    name: install Gramine
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+      - name: add gramine key
+        run: |
+               sudo curl -fsSLo /usr/share/keyrings/gramine-keyring.gpg https://packages.gramineproject.io/gramine-keyring.gpg
+               echo "deb [arch=amd64 signed-by=/usr/share/keyrings/gramine-keyring.gpg] https://packages.gramineproject.io/ $(lsb_release -sc) main" |  sudo tee /etc/apt/sources.list.d/gramine.list
+      - uses: awalsh128/cache-apt-pkgs-action@latest
+        with:
+          packages: rustc cargo gramine cmake clang gramine
+          version: 1.1
+          execute_install_scripts: true
+
+      - name: Set PATH
+        run: echo "export PATH=\$PATH:/usr/local/bin:/usr/bin" >> $GITHUB_ENV
+      - name: generate manifest and sig
+        run: |
```

Comment:
Does it make sense to move this into a (checked-in) bash script?
-> easier to debug
-> reusable
Or will nobody else ever want to run this?